stories/Checkbox-stories.tsx

```diff
@@ -100,19 +100,41 @@ export const LoadingState: Story = {
   render: (args) => <CheckboxState initialValue={args.checked} {...args} />,
 };
 
-function CheckboxState({
-  initialValue,
-  ...checkboxProps
-}: Partial<ComponentProps<typeof Checkbox>> & {
+export const WithTooltip: Story = {
+  args: {
+    checked: false,
+    label: "I'm a checkbox",
+    helpText: 'I have some help text',
+    onCheck: () => {},
+  },
+  render: (args) => (
+    <div style={{ display: 'flex', alignItems: 'center' }}>
+      <Tooltip content="test">
+        <div>
+          <CheckboxState initialValue={args.checked} {...args} />
+        </div>
+      </Tooltip>
```

Comment:
I think we should remove the wrapping divs, and just target the checkbox with the tooltip.
The tooltip trigger should always be an interactive element, so it makes sense to target the checkbox itself.